self:
  id: 1
  ip: 127.0.0.1
remote:
  - id: 2
    ip: 192.168.31.111
    send_port: 30001 # 可以相同
    recv_port: 40001 # 不能相同
    send_topics:
      - /robot2/cmd_vel #geometry_msgs::Twist
      - /robot2/cat_cmd #std_msgs::Bool
      - /robot2/motor_adj  #�ӻ��ĵ��������Ϣ
      - /robot2/mag_cmd  #�ӻ��ĵ������������
    recv_topics:
      - /robot2/Odometry #nav_msgs::Odometry
      - /robot2/cat_state #std_msgs::Bool
      - /robot2/imu_topic #�ӻ���IMU��Ϣ
      - /robot2/mag_state #�ӻ��ĵ����������Ϣ
  # - id: 3
  #   ip: 192.168.31.113
  #   send_port: 30001
  #   recv_port: 40002
  #   send_topics:
  #     - /robot3/cmd_vel #geometry_msgs::Twist
  #     - /robot3/cat_cmd #std_msgs::Bool
  #   recv_topics:
  #     - /robot3/Odometry #nav_msgs::Odometry
  #     - /robot3/cat_state #std_msgs::Bool